src/libraries/System.IO.Pipes.AccessControl/src/System.IO.Pipes.AccessControl.csproj

```diff
@@ -1,20 +1,26 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AssemblyName>System.IO.Pipes.AccessControl</AssemblyName>
-    <IncludeDefaultReferences>false</IncludeDefaultReferences>
     <IsPartialFacadeAssembly Condition="'$(TargetsWindows)' == 'true'">true</IsPartialFacadeAssembly>
     <OmitResources Condition="'$(TargetsWindows)' == 'true'">true</OmitResources>
     <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetsWindows)' != 'true'">SR.PlatformNotSupported_AccessControl</GeneratePlatformNotSupportedAssemblyMessage>
-    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;netcoreapp2.1;$(NetCoreAppCurrent)</TargetFrameworks>
     <Nullable>enable</Nullable>
   </PropertyGroup>
+  <PropertyGroup>
+    <IncludeDefaultReferences Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">false</IncludeDefaultReferences>
+    <AssemblyVersion Condition="'$(TargetFramework)' == 'netcoreapp2.1'">4.0.3.0</AssemblyVersion>
+  </PropertyGroup>
   <ItemGroup>
     <Reference Include="System.Runtime" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetsWindows)' == 'true'">
     <ProjectReference Include="..\..\System.IO.Pipes\src\System.IO.Pipes.csproj" />
     <ProjectReference Include="..\..\System.Security.AccessControl\src\System.Security.AccessControl.csproj" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.1'">
+    <Reference Include="System.Resources.ResourceManager" />
+  </ItemGroup>
```

Comment:
@ericstj the changes made in this file allowed this to pass:
`dotnet msbuild  D:\runtime\src\libraries\pkg\test\testPackages.proj /p:TestPackages=System.IO.Pipes.AccessControl`

Without these changes, we were seeing this error when running that test command:
```
D:\runtime\artifacts\bin\testPkg\support\packageTest.targets(49,5): error : Assembly 'System.IO.Pipes.AccessControl' is missing dependency 'System.Security.AccessControl' [D:\runtime\artifacts\bin\testPkg\projects\System.IO.Pipes.AccessControl\netcoreapp2.1\project.csproj] [D:\runtime\src\libraries\pkg\test\testPackages.proj]
D:\runtime\artifacts\bin\testPkg\support\packageTest.targets(49,5): error : Assembly 'System.IO.Pipes.AccessControl' is missing dependency 'System.Security.Principal.Windows' [D:\runtime\artifacts\bin\testPkg\projects\System.IO.Pipes.AccessControl\netcoreapp2.1\project.csproj] [D:\runtime\src\libraries\pkg\test\testPackages.proj]
```